README.md

```diff
@@ -34,6 +34,7 @@ __NOTE__: It's early days so please be gentle when reporting issues :) As author
 
         post 'creates a new blog' do
           consumes 'application/json'
+          operation_description 'Detailed implementation notes for the create a new blog API endpoint'
```

Comment:
I'd like to name this "description" rather than "operation_description" to be consistent with the other properties (consumes, produces etc.). The fact that they apply to an operation is implied by their position within the "post" context. My goal is for the terms and general structure of the DSL to align closely to those used by the swagger-spec itself, so that someone fluent in Swagger can pick up the DSL very quickly.